TextField/index.d.ts

```diff
@@ -2,23 +2,23 @@ import MaterialComponent from '../MaterialComponent';
 import { VNode } from 'preact';
 import { MDCFoundation, MDCComponent, MDCRipple } from '../MaterialComponentsWeb';
 
-declare interface IHelptextProps extends JSX.HTMLAttributes {
+declare interface IHelperTextProps extends JSX.HTMLAttributes {
   persistent?: boolean;
   'validation-msg'?: boolean;
 }
-declare class Helptext extends MaterialComponent<IHelptextProps, {}> {}
+declare class HelperText extends MaterialComponent<IHelperTextProps, {}> {}
 
 declare interface ITextFieldProps extends JSX.HTMLAttributes {
-  helptext?: string;
-  helptextPersistent?: string;
-  helptextValidationMsg?: string;
+  helperText?: string;
+  helperTextPersistent?: string;
+  helperTextValidationMsg?: string;
```

Comment:
Maybe not for this PR but aren't we missing here some props @YePpHa ?
https://github.com/prateekbh/preact-material-components/blob/master/TextField/TextField.jsx#L125

Reply:
Ok, PR closed 👍  
It can be taken care later